docs/manual/install.qmd

```diff
@@ -69,13 +69,18 @@ the package.
 
 To generate the documentation, you'll first have to:
 
-- Install [Quarto](https://quarto.org/docs/get-started/)
-- Install Python dependencies:
+1. Install [Quarto](https://quarto.org/docs/get-started/)
+2. Install Python dependencies:
 
     ```bash
     pip install -r requirements-doc.txt
     ```
 
+3. Run `build-api.sh`^[This is currently not very portable, and requires a
```

Comment:
Mention that this has to be run while within the `docs/manual` directory and you have to be within the virtual environment if you're using it.

Reply:
path added, but I don't think it's necessary to mention the virt. env. since that's one of the previous steps in the numbered list?

Reply:
path added, but I don't think it's necessary to mention the virt. env. since that's one of the previous steps in the numbered list?